packages/muelu/doc/UsersGuide/paramlist_hidden.tex

```diff
@@ -234,13 +234,21 @@
         
 \cbb{rap: triple product}{bool}{false}{Use all-at-once triple matrix product kernel}
         
-\cba{matrixmatrix: kernel params}{\parameterlist}{Controls the kernel algorithms for matrix-matrix operations}
-        
```

Comment:
@DrBooom Did you modify this file by hand?  The proper way is to remove the option from muelu/doc/UsersGuide/masterlist.xml, then run `update_params.sh` in that same directory.  This will update MueLu's validator and users guide simultaneously.